language/en

```diff
@@ -201,5 +201,17 @@
 			"remove_title": "Remove Photo",
 			"confirm_remove": "Are you sure you want to remove this Photo?"
 		}
-	}
+	},
+	"draw_tip": "Draw on Canvas",
+	"add_text": "add new text to canvas",
```

Comment:
@RukayaDavids Why with some you start capital letter and others you do not ?

Reply:
@alcinovanrooyen I added the value's from the html in our demo page. Please let me know how I should edit it.

Reply:
@RukayaDavids which demo page ?

Reply:
Any one but you can look at this one https://wp.demo.pitchprint.io/product/blue-business-card/. When you right click the elements it shows the words like that.

Reply:
@alcinovanrooyen please look at this one https://wp.demo.pitchprint.io/product/blue-business-card/.

Reply:
@RukayaDavids I will close this pull request but not merge it, as the file has been changed by Isaac on server, will download the file and manually add your updates from this pull request to it. Thanks for this.